test-site/src/components/HTMLExampleCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { CardProps, useCardAnalyticsCallback, useCardFeedbackCallback } from '@yext/search-ui-react';
+import { useCallback } from 'react';
+import '../styles/resetStyles.css'
+
+const builtInCssClasses = {
+    container: 'flex flex-col justify-between border border-gray-200 rounded-lg mb-4 p-4 shadow-sm',
+    header: 'flex text-neutral-dark',
+    title: 'text-lg font-medium',
+    thumbsFeedbackContainer: 'flex justify-end mt-4 text-sm text-gray-500 font-medium',
+};
+
+// change to the field name that contains html string
+const htmlFieldName = "c_richTextV2Data";
+interface CustomRawDataType {
+    name: string,
```

Comment:
maybe call it `title`?

Reply:
Do you mean changing "htmlFieldName" to title? I think htmlFieldName might be more flexible because HTML can be returned for any field that has converted RichTextV2/markdown.